jte/pom.xml

```diff
@@ -24,6 +24,16 @@
             <artifactId>jte-runtime</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>gg.jte</groupId>
+            <artifactId>jte-extension-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Should we remove the native resources extension here? This way users who need it can add the dependency and others don't have it by default?

Reply:
The intention was to keep backward compatibility for users who have configured native resources with the existing boolean property. Maybe there is another way to do that. Do you have any ideas?

I can figure out how to do it in Gradle :)

Reply:
Maybe we could throw an Exception when the native resources boolean property is set to true? And put instructions into the exception message how to migrate to the new extension?

Reply:
I guess if we put this in 3.0 we can just drop the existing setting and be less concerned about compatibility.

Reply:
Yes, that makes sense 👍